heareval/embeddings/task_embeddings.py

```diff
@@ -204,16 +204,13 @@ def get_labels_for_timestamps(labels: List, timestamps: np.ndarray) -> List:
     # A list of labels present at each timestamp
     timestamp_labels = []
 
-    # TODO: make sure we are saving the labels in the correct units.
-    #   dcase gives units in seconds whereas the timestamps from the API
-    #   are in millseconds. Make sure dataset events are specified in ms.
+    # NOTE: Make sure dataset events are specified in ms.
```

Comment:
Yes -- we should include this in a README somewhere for when we are creating more pipelines.

Reply:
https://github.com/neuralaudio/hear-eval-kit/issues/138